monitor/census.go

```diff
@@ -306,8 +306,8 @@ func InitCensus(nodeType NodeType, version string) {
 	census.mTicketValueSent = stats.Float64("ticket_value_sent", "TicketValueSent", "gwei")
 	census.mTicketsSent = stats.Int64("tickets_sent", "TicketsSent", "tot")
 	census.mPaymentCreateError = stats.Int64("payment_create_errors", "PaymentCreateError", "tot")
-	census.mDeposit = stats.Float64("broadcaster_deposit", "Current remaining deposit for the broadcaster node", "gwei")
-	census.mReserve = stats.Float64("broadcaster_reserve", "Current remaining reserve for the broadcaster node", "gwei")
+	census.mDeposit = stats.Float64("gateway_deposit", "Current remaining deposit for the gateway node", "gwei")
```

Comment:
@thomshutt @pwilczynskiclearcode @hjpotter92 Any thoughts if we're using these metrics anywhere (grafana, alerts, etc.) and they'd need to be updated?

Reply:
@leszko, @thomshutt, I believed I had implemented it in a non-breaking way 🤔. What changes would you like me to make?